internal/resources/sourcesecret/data_sourcesecret_test.go

```diff
@@ -0,0 +1,125 @@
+/*
+Copyright © 2022 VMware, Inc. All Rights Reserved.
+SPDX-License-Identifier: MPL-2.0
+*/
+
+package sourcesecret
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	policyoperations "github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/operations"
+	"github.com/vmware/terraform-provider-tanzu-mission-control/internal/resources/policy/scope"
```

Comment:
Please remove all references to policy from the tests, if there is something that is common please move it to the helpers to reuse across the codebase.

Reply:
Update PTAL